starter-code/index.html

```diff
@@ -9,6 +10,60 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
+
 <body>
+  <h1>Ironhack Merchandising Shop</h1>
+  <div id="products">
+    <div class="product-line" id="hidden">
+      <div class="product-name">
+        <span>Ironhack Surfboard</span>
+      </div>
+      <div class="product-price price">
+        <p>$</p><span class="unit-price">25.00</span>
+      </div>
+      <div class="quantity">
+        <label>QTY</label>
+        <input type="number" class="unit-qty">
```

Comment:
Little tip, if you don't want to allow negative values, do:
```
<input type="number" class="unit-qty" min="0">
```
Keep in mind this is not a perfect solution.